### ISSUE TITLE
Doesnt work with Sketch 3.3.1

### ISSUE THREAD

**Issue by jarisanders** [ddwht/sketch-dynamic-button#15]
Running the plugin creates an empty folder and a shape that I'm unable to find back in my layers list.

**Comment by alekspetrov** [comment]
I have the same issue =(

**Comment by shenjenny** [comment]
Also using 3.3.1. it creates a folder called flex button but the button shape is unchanged.

**Comment by zyltd1990** [comment]
same here

**Comment by colbydodson** [comment]
same here

**Comment by jamesgrubb** [comment]
Same issue here

**Comment by dmulranen** [comment]
Sadness :(

**Comment by justinkuntz** [comment]
You can still go the manual method and it works (I'm running 3.3.1). Create a folder structure with type layer and a layer with the button shape called "BG" under. Use the shortcut Cmd + J on the type layer. You will see the text layer with (0:0:0:0). Change the padding and hit Cmd + J again the button with resize.

**Comment by shenjenny** [comment]
nope, I tried that and the button didn't resize. Just created an empty folder called flex button. (Using 3.3.2 now)

**Comment by ivostork** [comment]
Thx justinkuntz this trick is working for me (3.3.2).

**Comment by megaroeny** [comment]
@justinkuntz That works in 3.3.2 also! Thanks

**Comment by jlmakes** [comment]
I uninstalled and reinstalled the plugin. It works perfectly in `Sketch 3.3.2`

**Comment by realjoet** [comment]
Uninstall and reinstall working on 3.3.3 as well! @justinkuntz++ for the tip!!

**Comment by duduct** [comment]
Same issue for 3.3

**Comment by jhebb** [comment]
Uninstalling and reinstalling worked for me in 3.4.1 - thanks!
